docs/_docs/guides/open-source-contributors.md

```diff
@@ -242,7 +242,20 @@ For example to create Demo data, you can now run: `mojito demo-create -n DemoCLI
 Alternatively, install the CLI using the [install scripts]({{ site.url }}/docs/guides/install/#cli-install-script).
 
 ## Run Unit Tests
-   
+
+
+### Testcontainers
+By default, if no datasource is configured in the application.properties file, integration tests rely on 
```

Comment:
One of the advantage of the current setup is that everything can be running out of the box for simple development. Adding docker as requirement adds complexity and might flakiness/impact perf. I'd keep the default settings to run in memory and just activate container testing with a spring profile that we can use or not locally and in dev. On current CI we could run both tests in parallel at the beginning?

Reply:
Given that other popular Java frameworks now rely on similar dependencies for testing (e.g.: Quarkus) - I don't have any specific reason to believe that flakiness/perf is going to be significantly impacted by this change.

The biggest value for integration tests is that they find issues locally, before deploying the application live. I believe ensuring the default behaviour is as close to production and finds as many bugs should be "base" state then. (opt-out vs opt-in).

If the target is simplifying setup - we could make the existing setup steps significantly simpler by removing the MySQL install, configuration & application configuration setup steps and replacing that with the requirement to just have Docker instead (+ also changing the default for the local application to use TC). 
I would expect it's more likely today to have Docker already installed on one's machine than to find the right version of MySQL that this application depends on - but I wouldn't have any data backing that.